Selenium/EmployeeRegister.py

```diff
@@ -0,0 +1,132 @@
+        click_button(driver, By.ID, "NavbarRestaurantsSectionButton")
+
+        # Czekamy na zmianę strony
+        wait_for_url_to_be(driver, restaurants_management_url)
+        click_button(driver, By.ID, "menu-listItem-employees-button")
+
+        # czekamy na załadowanie tabelki
+        # to do
+
+        # przechodzimy do fromularza rejestracji pracownika
+        # czy dla tego id nie powinna byc zmieniona nazwa?
+        click_button(driver, By.ID, "RestaurantListAddRestaurantButton")
+
+        # Sprawdzenie czy pojawia się nowa zawartość
+
+        #firstname
+        click_text_field(driver, By.ID, "firstName")
+
+        press_tab_key(driver)
+
+        wait_for_element(driver, By.ID, "errorMes-wrap")
+        #czy sa id dla error message bo narazie dla wszystkich widze: "errorMessage-wrap"
+
+        #lastname
+        click_text_field(driver, By.ID, "lastName")
+
+        press_tab_key(driver)
+
+        wait_for_element(driver, By.ID, "errorMes-wrap")
+
+        #login
+
+        click_text_field(driver, By.ID, "login")
+
+        press_tab_key(driver)
+
+        wait_for_element(driver, By.ID, "errorMes-wrap")
+
+        #phone
+        #nie ma ID
+        click_text_field(driver, By.NAME, "phoneNumber")
+
+        press_tab_key(driver)
+
+        wait_for_element(driver, By.ID, "errorMes-wrap")
+
+        #password
+
+        click_text_field(driver, By.ID, "password")
+
+        press_tab_key(driver)
+
+        wait_for_element(driver, By.ID, "errorMes-wrap")
+
+        #password_confirmation
+        click_text_field(driver, By.ID, "confirmPassword")
+
+        press_tab_key(driver)
+
+        wait_for_element(driver, By.ID, "errorMes-wrap")
+
+
+        # Znaleznienie pola i wypełnienie go danymi
+
+        #selector value to ID?
+        enter_text(driver, By.ID, "firstName", sample_data.first_name)
+
+        enter_text(driver, By.ID, "lastName", sample_data.last_name)
+
+        enter_text(driver, By.ID, "login", sample_data.login)
+
+        enter_text(driver, By.NAME, "phoneNumber", sample_data.phone)
+
+        enter_text(driver, By.ID, "password", sample_data.password)
+
+        enter_text(driver, By.ID, "confirmPassword", sample_data.password_confirmation)
+
+        wait_for(delay)
+
+        universal_wait_for(driver, EC.element_to_be_clickable, By.ID, "EmployeeRegisterSubmitButton")
+
+        click_button(driver, By.ID, "EmployeeRegisterSubmitButton")
+
+        # TODO sprawdzic czy test przeszedł
+        return True  # Test przeszedł
+
+    except Exception as e:
+        result(str(e), False)
+        info("Register restaurant test failed")
+        info(f"Current URL: {driver.current_url}")
+        info(f"Page title: {driver.title}")
+        print("-----------------------")
+        info("Problem with following login data:")
+        info(sample_data)
+        return False  # Test nieudany
+
+    finally:
+        wait_for(delay)
+
+
+if __name__ == "__main__":
```

Comment:
Generalnie staramy się nie duplikować mainów, mamy stworzony main.py w projekcie i to służy nam za maina, więc można przenieść wywołanie testu do tamtego maina, a tego wykasować (żeby w tamtym test działał, trzeba go zaimportować, na górze kodu maina są już przykładowe importy)

Reply:
Akurat wydaje mi się że main może się tutaj przydać żeby wywołać test osobno i nie uruchamiać wszystkich na raz przez main.py, ale to nie ma większej różnicy

Reply:
W sumie, też racja, main też może zostać